spec/display-buffer-spec.coffee

```diff
@@ -724,6 +724,15 @@ describe "DisplayBuffer", ->
         expect(displayBuffer.clipScreenPosition([0, 1], clip: 'forward')).toEqual [0, tabLength]
         expect(displayBuffer.clipScreenPosition([0, tabLength], clip: 'forward')).toEqual [0, tabLength]
 
+  describe "::screenPositionForPixelPosition(pixelPosition)", ->
+    it "pixel positions above buffer to map to start", ->
+      expect(displayBuffer.screenPositionForPixelPosition(top: -Infinity, left: -Infinity)).toEqual [0, 0]
+      expect(displayBuffer.screenPositionForPixelPosition(top: -Infinity, left: Infinity)).toEqual [0, 0]
+
+    it "pixel positions below buffer map to end", ->
```

Comment:
:bulb: Do you think we could make these specs a bit more descriptive? I was thinking to something like:

```coffee
it "clips pixel positions above buffer start"
it "clips pixel positions below buffer end"
```

Maybe we could even add some more expectations, just to make clear that we don't treat `Infinity` as a special case, e.g.

```coffee
expect(displayBuffer.screenPositionForPixelPosition(top: -1, left: Infinity)).toEqual [0, 0]
expect(displayBuffer.screenPositionForPixelPosition(top: -1, left: 4)).toEqual [0, 0]
```

Nice job here, by the way :wink: